requirements/docker.txt

```diff
@@ -9,6 +9,7 @@
 -e file:.                 # via -r requirements/base.in
 gevent==20.6.2            # via -r requirements/docker.in
 greenlet==0.4.16          # via gevent
+psycopg2-binary==2.8.6    # via -r requirements/docker.in
```

Comment:
My understanding is that the Docker build can't contain `psycopg2` due to licensing.

Reply:
This is only used to build Docker, which isn't actually pushed anywhere.